### PR TITLE
Include Navbar

### PR DESCRIPTION
**Pull request by RolandNaijuka**
Include navbar and change the backgroundcolor to rgb(73, 67, 67)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,3 @@
 # Google Student Training in Engineering Program
-This repo contains Roland Naijuka's portfolio and STEP projects.
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
+This repo contains Roland Naijuka's portfolio as part of the  STEP  Program at Google.
 
-To get started:
```
Comment (vitashubin) [inline README.md:6]: might be useful to leave the get started info for future reference, but up to you!

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,56 @@
...
+}
+
+#navbar .navbar-item{
+    padding:8px 16px;
+    float:left;
+    width:auto;
+    border:none;
+    display:block;
+    outline:0s;
+}
```
Comment (vitashubin) [inline portfolio/src/main/webapp/style.css:19]: nit: alphabetize the fields 
e.g.:
border
display
float
....

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,56 @@
...
+#nav-right-side{
+    float: right;
+}
+
+.clear{
+    clear: both;
+}
+
 #content {
-  margin-left: auto;
```
Comment (vitashubin) [inline portfolio/src/main/webapp/style.css:2]: nit: I believe styleguide convention here would be to do "margin: 0px auto"
Same for the margins below

File: walkthroughs/week-1-setup/intro-walkthrough.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ Wherever you see a file that ends with `-walkthrough.md`, you can run the
 this command runs the walkthrough you're currently reading:
 
 ```bash
-teachme ~/step/walkthroughs/week-1-setup/intro-walkthrough.md
+
```
Comment (vitashubin) [inline walkthroughs/week-1-setup/intro-walkthrough.md:97]: Did you mean to remove just this single line?

**Review by vitashubin — COMMENTED** [review]
Do you have a demo link we could go to? :) 

Code looks great!

File: README.md (hunk 1 above)
Comment (RolandNaijuka) [reply README.md:6]: > might be useful to leave the get started info for future reference, but up to you!

I made these changes in PR #6

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (RolandNaijuka) [reply portfolio/src/main/webapp/style.css:19]: > nit: alphabetize the fields
> e.g.:
> border
> display
> float
> ....

I made these changes in PR #6

File: walkthroughs/week-1-setup/intro-walkthrough.md (hunk 4 above)
Comment (RolandNaijuka) [reply walkthroughs/week-1-setup/intro-walkthrough.md:97]: > Did you mean to remove just this single line?

This was an error. I corrected it in #6

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (RolandNaijuka) [reply portfolio/src/main/webapp/style.css:2]: > nit: I believe styleguide convention here would be to do "margin: 0px auto"
> Same for the margins below

Thank you that comment. I have made the necessary changes

**Comment by RolandNaijuka** [comment]
> Do you have a demo link we could go to? :)
> 
> Code looks great!

I will send this in the next PR

**Pull request closed without merging**